### PR TITLE
Add unit test cases for vote revocation when validators are ineligible

### PR DESCRIPTION
**Pull request by eelanagaraj**
### Description

We saw a bug in kliento when trying to revoke votes from an ineligible validator. While trying to debug that and understand what was going on in `Elections.sol`, I couldn't find unit test cases for that specific case so went ahead and added them. (More than anything, I think it'll be useful documentation of how to call the revoke functions in that edge case (`lesser` and `greater` are not used and don't matter in that case), and make it easier to understand what's going on).

### Tested

- Just adds tests.

### CODE REVIEW

**Review by soloseng — APPROVED** [review]
🚀

File: packages/sdk/contractkit/src/wrappers/Election.test.ts [hunk 1]
```diff
@@ -19,6 +19,8 @@ const blsPublicKey =
 const blsPoP =
   '0xcdb77255037eb68897cd487fdd85388cbda448f617f874449d4b11588b0b7ad8ddc20d9bb450b513bb35664ea3923900'
 
+jest.setTimeout(20000)
```
Comment (eelanagaraj) [inline packages/sdk/contractkit/src/wrappers/Election.test.ts:22]: this should fix the test flakiness (does so locally at least), looks like this is caused by a timeout on the findLesserGreater tests 🤞

File: packages/sdk/contractkit/src/wrappers/Election.test.ts [hunk 2]
```diff
@@ -146,15 +168,25 @@ testWithGanache('Election Wrapper', (web3) => {
...
         }
         await Promise.all(promises)
+      })
 
+      test('SBAT activate vote', async () => {
+        const activeVotes = await election.getActiveVotesForGroup(groupAccount)
+        expect(activeVotes).toEqual(ONE_HUNDRED_GOLD)
+      })
+
+      test('active votes remain unchanged when group becomes ineligible', async () => {
```
Comment (soloseng) [inline packages/sdk/contractkit/src/wrappers/Election.test.ts:178]: Should the validator be deactivated in this test?

Comment (eelanagaraj) [reply packages/sdk/contractkit/src/wrappers/Election.test.ts:178]: yes good catch!
